src/Symfony/Component/HttpFoundation/Session/Storage/MockArraySessionStorage.php

```diff
@@ -75,8 +73,6 @@ public function __construct($name = 'MOCKSESSID', MetadataBag $metaBag = null)
     }
 
     /**
-     * Sets the session data.
```

Comment:
But then the `@param` below can be removed as well... and thousands of other phpdocs. Let's not do that. A good time will be when we can really get rid of the vast majority of current phpdocs (but on all the code base at once). Right now, that would a lot of work that does not beings value and will give the core team a lot of work when merging branches. So, I'm 👎 for these changes.